lib/cts/cases/cli.rb

```diff
@@ -149,10 +152,18 @@ def create(*target_states)  # rubocop:disable Metrics/CyclomaticComplexity, Metr
       if options.key? :received_date
         CTS.info "Setting received date to: #{options[:received_date]}"
       end
+      if options.key? :creator
```

Comment:
oops - you call it :user in your options declaration, but its :creator here. I think we might need some more tests in this area?

Reply:
Good spot